Submodels/MTSOL_call.py

```
@@ -1,20 +1,613 @@
+                self.process.stdin.flush()
+
+                # Increase iteration counter by step size
+                iter_counter += self.ITER_STEP_SIZE           
+
+        # Return the exit flag and iteration counter
+        return exit_flag, iter_counter
+
+
+    def GetAverageValues(self,
+                         file_name: str = 'forces',
+                         skip_lines: dict = {2, 14, 26, 27, 32, 37, 42},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)